tsconfig.base.json

```diff
@@ -16,6 +16,7 @@
     "noUnusedLocals": true,
     "importHelpers": true,
     "strictNullChecks": true,
+    "stripInternal": true,
```

Comment:
Why this change? I missed anything in this PR that needs this.

Reply:
We have one field somewhere that's marked as `@internal`, this removes it from the exported package types. I think that comes with the AppProvider component though, I broke the PR up in two and this landed in the wrong place. We'll still need it though.